flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-oceanbase/src/main/java/org/apache/flink/cdc/connectors/oceanbase/sink/OceanBaseEventSerializationSchema.java

```diff
@@ -0,0 +1,141 @@
+        pipelineZoneId = zoneId;
+    }
+
+    @Override
+    public Record serialize(Event event) {
+        if (event instanceof DataChangeEvent) {
+            return applyDataChangeEvent((DataChangeEvent) event);
+        } else if (event instanceof SchemaChangeEvent) {
+            SchemaChangeEvent schemaChangeEvent = (SchemaChangeEvent) event;
+            TableId tableId = schemaChangeEvent.tableId();
+            if (event instanceof CreateTableEvent) {
+                schemaMaps.put(tableId, ((CreateTableEvent) event).getSchema());
+            } else {
+                if (!schemaMaps.containsKey(tableId)) {
+                    throw new RuntimeException("schema of " + tableId + " is not existed.");
+                }
+                schemaMaps.put(
+                        tableId,
+                        SchemaUtils.applySchemaChangeEvent(
+                                schemaMaps.get(tableId), schemaChangeEvent));
+            }
+        }
+        return null;
+    }
+
+    private Record applyDataChangeEvent(DataChangeEvent event) {
+        TableId tableId = event.tableId();
+        Schema schema = schemaMaps.get(tableId);
+        Preconditions.checkNotNull(schema, event.tableId() + " is not existed");
+        Object[] values;
+        OperationType op = event.op();
+        boolean isDelete = false;
+        switch (op) {
+            case INSERT:
+            case UPDATE:
+            case REPLACE:
+                values = serializerRecord(event.after(), schema);
+                break;
+            case DELETE:
+                values = serializerRecord(event.before(), schema);
+                isDelete = true;
+                break;
+            default:
+                throw new UnsupportedOperationException("Unsupported Operation " + op);
+        }
+        return buildDataChangeRecord(tableId, schema, values, isDelete);
+    }
+
+    private DataChangeRecord buildDataChangeRecord(
+            TableId tableId, Schema schema, Object[] values, boolean isDelete) {
+        Preconditions.checkState(
+                Objects.nonNull(tableId.getSchemaName()), "Schema name cannot be null or empty.");
+        com.oceanbase.connector.flink.table.TableId oceanBaseTableId =
+                new com.oceanbase.connector.flink.table.TableId(
+                        tableId.getSchemaName(), tableId.getTableName());
+        TableInfo tableInfo =
+                new TableInfo(
+                        oceanBaseTableId,
+                        schema.primaryKeys(),
+                        schema.getColumnNames(),
+                        Lists.newArrayList(),
+                        null);
+
+        return new DataChangeRecord(
+                tableInfo,
+                isDelete ? DataChangeRecord.Type.DELETE : DataChangeRecord.Type.UPSERT,
+                values);
+    }
+
+    /** serializer RecordData to oceanbase data change record. */
+    public Object[] serializerRecord(RecordData recordData, Schema schema) {
```

Comment:
Please add a test for this method to ensure that add support types are converted as expected (especially for timestamp related type).

Reply:
It will be covered by the newly added test `org.apache.flink.cdc.connectors.oceanbase.sink.OceanBaseEventSerializationSchemaTest#testMixedSchemaAndDataChanges`.